task_manager/spec/factories/users.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :user do
+    name { 'MyString' }
+    password { 'MyString' }
+    password_confirmation { 'MyString' }
+    sequence :email do |n|
+      "test#{n}@example.com"
+    end
```

Comment:
【修正任意】
下記のような書き方もできますー
あくまで好みの問題なので修正は任意です。好みの方を使ってください:pray:

```suggestion
    sequence (:email, "test_1@example.com")
    # => test_1@example.com, test_2@example.com, test_3@example.com
```

Reply:
そんな簡単にかけるんですね、、、
ありがとうございます！そっちで書いてみます！

Reply:
修正しました！ご確認おねがいします！
https://github.com/Fablic/training/pull/1096/commits/53163a56fe76bd7babcf01305e64220970a51e8c